packages/format-library/src/link/use-link-instance-key.js

```diff
@@ -17,6 +17,9 @@ function getKey( _id ) {
  * @return {string} the unique key to use for this link control.
  */
 function useLinkInstanceKey( instance ) {
+	if ( ! instance ) {
```

Comment:
This can be `null` or a `anchorRef`. So we need to avoid the null case.

Reply:
Could we add a comment above to explicitly state that we're returning a `undefined` here on purpose?

Reply:
Apologies @kevin940726, I just missed your comment here before I hit "merge". I was perhaps over-eager to get this fix in, in time for the 11.9 release!

Reply:
No worries 👍 , it's more like a nitpick anyway :).